runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -7885,8 +7885,65 @@ done:;
 	VMINLINE VM_BytecodeAction
 	defaultvalue(REGISTER_ARGS_LIST)
 	{
+retry:
+		j9object_t newObject = NULL;
 		VM_BytecodeAction rc = EXECUTE_BYTECODE;
-		// TODO: Implement bytecode
+		U_16 index = *(U_16*)(_pc + 1);
+		J9ConstantPool *ramConstantPool = J9_CP_FROM_METHOD(_literals);
+		J9RAMClassRef *ramCPEntry = ((J9RAMClassRef*)ramConstantPool) + index;
+		J9Class* volatile resolvedClass = ramCPEntry->value;
+
+		if ((NULL != resolvedClass) && (resolvedClass->romClass->modifiers & J9AccValueType)) {
+
+			if (!VM_VMHelpers::classRequiresInitialization(_currentThread, resolvedClass)) {
+				j9object_t instance = _objectAllocate.inlineAllocateObject(_currentThread, resolvedClass);
+				if (NULL == instance) {
+					updateVMStruct(REGISTER_ARGS);
+					instance = _vm->memoryManagerFunctions->J9AllocateObject(_currentThread, resolvedClass, J9_GC_ALLOCATE_OBJECT_INSTRUMENTABLE);
+					VMStructHasBeenUpdated(REGISTER_ARGS);
+					if (J9_UNEXPECTED(NULL == instance)) {
+						rc = THROW_HEAP_OOM;
+						goto done;
+					}
+				}
+				newObject = instance;
+				goto done;
+			}
+			/* Class requires initialization */
+			buildGenericSpecialStackFrame(REGISTER_ARGS, 0);
+			updateVMStruct(REGISTER_ARGS);
+			initializeClass(_currentThread, resolvedClass);
+			VMStructHasBeenUpdated(REGISTER_ARGS);
+			restoreGenericSpecialStackFrame(REGISTER_ARGS);
+			if (immediateAsyncPending()) {
+				rc = GOTO_ASYNC_CHECK;
+				goto done;
+			} else if (VM_VMHelpers::exceptionPending(_currentThread)) {
+				rc = GOTO_THROW_CURRENT_EXCEPTION;
+				goto done;
+			}
+			goto retry;
+		}
+
+		/* Unresolved */
```

Comment:
Add more detail to this comment. What is unresolved, and why we need to resolve it?